technology-considerations/API_management_AppSync_GraphQL_REST.md

```diff
@@ -0,0 +1,62 @@
+# Options for API management
+
+We want to have an intermediate component between our frontend (which consists of the browser and the Play web application)
+and the database. 
+This component will represent our APIs. The APIs will include virus scanning, format checking, validations of the format.
+
+The APIs can use the [REST architectural style]  (https://en.wikipedia.org/wiki/Representational_state_transfer),
+or we could also take advantage of [GraphQL](https://graphql.org/), that introduces a schema and a type system, along with greater flexibility.
+
+We decided to explore using GraphQL for the following reasons:
+- we anticipate multiple datasources, a relational database for keeping the information about files, transfers, users 
```

Comment:
Is the API actually going to get data from the auditing system, or is going to be quite a separate system?

Reply:
Not decided yet, we want to keep both options, I added this point (if we decide to put the auditing system in TDR rather than separating it into  a different system)